coq-event-struct.opam

```diff
@@ -17,7 +17,7 @@ build: [make "-j%{jobs}%" ]
 install: [make "install"]
 depends: [
   "coq" {(>= "8.10" & < "8.13~") | (= "dev")}
-  "coq-mathcomp-finmap" {(= "dev")}
+  "coq-mathcomp-finmap"
```

Comment:
You should specify the precise minimum version for `coq-mathcomp-finmap`.

Reply:
Ok, I see